.gitignore

```
@@ -10,3 +10,4 @@ __pycache__/
 version
 .ruff_cache/
 _build
+slurm-charms.auth
```

What if we used a secret dotfile named `.secret` or `.charmhub.secret`? We could then configure GitHub to scream at us if this file were to be accidentally committed and pushed to the public repo. Also, I think "secret" slightly communicates better that this sensitive information that we don't want to expose.

Also [nit]: missing newline at the end of the file :sweat_smile: 

```suggestion
slurm-charms.auth

```